resources/keywords/import_provider_account.resource

```diff
@@ -284,6 +288,14 @@ User Imports ${isv} Provider Account Using CLI
 Provider Account Imported Successfully Using CLI
     Wait Until Keyword Succeeds    4x    5s    Describe Provider Account Using OC CLI
 
+User Navigates To Add ${databaseProvider} Topology Screen From Developer View
```

Comment:
@ikanias Duplicate - Keyword "User Navigates To Add Topology Screen From Developer View" available under import_provider_account.resource file, where the database provider can be send as a parameter like "_User Navigates To Add Topology Screen From Developer View    MongoDB_"

Reply:
OK got it. I see that the other function is:
User Navigates To Add Topology Screen From Developer View
    [Arguments]     ${databaseProvider}     ${ns}=${operatorNamespace}      ${developer}=False
    User Selects Developer View
    User Filters ${ns} Namespace On Project Dropdown
    User Navigates To Developers Catalog Add Screen
    IF  "${developer}"=="False"
        User Clicks On Cloud Hosted Database Menu
        User Selects ${databaseProvider} Tile Under Cloud Hosted Database
        Click Element    ${btnDBProvAddTopology}
    END

Is this the one we should keep?

Reply:
OK got you. fixed it to be as you described.